package.json

```diff
@@ -8,6 +8,7 @@
     ]
   },
   "dependencies": {
+    "@ant-design/icons": "^5.2.6",
```

Comment:
@siva9495 i don't think its right time to add this dependency. you can use the existing svgs for this. if you open the actions popup my pressing on the circle you will find the two icons. use them instead :) 
![image](https://github.com/tijlleenders/ZinZen/assets/39093015/7c4cf077-afa8-4174-847c-c302e9ec5e56)

Reply:
@Tushar-4781 I have a doubt regarding whether the SVG needs to have the same color or be white. Please confirm.

Reply:
its better to use the svgs like we are using in other components. i don't understand the sizing issue you mentioned. can you put some ss of code or output. thanks.

Reply:
@Tushar-4781 , sorted it. Removed the library and fixed it using SVGs for the icons. Can you please confirm?